### PR TITLE
area(Uninstall): Updating Uninstallation Steps

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

**What this PR does / why we need it**: Updating Uninstallation Steps

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [X] Signed the commit for DCO to be passed
-   [X] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/user-guides/uninstall-litmus.md [hunk 1]
```diff
@@ -27,36 +27,42 @@ To disconnect the [ChaosAgent](../getting-started/resources.md#chaosagents) conn
...
-To remove the respective components of the ChaosAgents you need to manually delete the created resources of that ChaosAgent.
+To remove the CRs Litmus uses, use the following command:
+
+- To remove individual CRs
+
+```bash
+kubectl delete chaosexperiments <CHAOSEXPERIMENTS_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete chaosresults <CHAOSRESULTS_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete workflows <WORKFLOW_NAMEs> -n <AGENT_NAMESPACE>
+kubectl delete cronworkflows <CRONWORKFLOW_NAMEs> -n <AGENT_NAMESPACE>
```
Comment (imrajdas) [inline website/docs/user-guides/uninstall-litmus.md:40]: There are a few more CDRs like event tracker...etc. Here is the link for all CRDs -https://github.com/litmuschaos/litmus/blob/master/litmus-portal/litmus-portal-crds.yml

File: website/docs/user-guides/uninstall-litmus.md [hunk 2]
```diff
@@ -27,36 +27,42 @@ To disconnect the [ChaosAgent](../getting-started/resources.md#chaosagents) conn
...
+- To remove all CRs
 
 ```bash
-kubectl delete chaosexperiments <CHAOSEXPERIMENTS_NAMEs> --all -<AGENT_NAMESPACE>
-kubectl delete chaosresults <CHAOSRESULTS_NAMEs> --all -<AGENT_NAMESPACE>
-kubectl delete workflows <WORKFLOW_NAMEs> --all -<AGENT_NAMESPACE>
-kubectl delete cronworkflows <CRONWORKFLOW_NAMEs> --all -<AGENT_NAMESPACE>
-kubectl delete deployment chaos-operator-ce chaos-exporter --all -<AGENT_NAMESPACE>
+kubectl delete chaosengine,chaosexperiments,chaosresults --all <AGENT_NAMESPACE>
+kubectl delete workflows cronworflows --all <AGENT_NAMESPACE>
```
Comment (imrajdas) [inline website/docs/user-guides/uninstall-litmus.md:48]: same steps for delete all other CRs like eventtracker ...etc

File: website/docs/user-guides/uninstall-litmus.md [hunk 3]
```diff
@@ -27,36 +27,42 @@ To disconnect the [ChaosAgent](../getting-started/resources.md#chaosagents) conn
...
+### Delete the Deployments
 
-To remove all the ChaosAgents component ever created from the system, apply this command.
+To remove the respective deployments of the ChaosAgents you need to manually delete them.
 
 ```bash
-kubectl delete chaosengine,chaosexperiments,chaosresults --all -A
-kubectl delete workflows cronworflows --all -<AGENT_NAMESPACE>
-kubectl delete deployment chaos-operator-ce chaos-exporter --all -A
+kubectl delete deployment chaos-operator-ce event-tracker workflow-controller chaos-exporter -n <AGENT_NAMESPACE>
```
Comment (imrajdas) [inline website/docs/user-guides/uninstall-litmus.md:57]: subscriber deployment is left

**Review by imrajdas — CHANGES_REQUESTED** [review]

File: website/docs/user-guides/uninstall-litmus.md (hunk 1 above)
Comment (S-ayanide) [reply website/docs/user-guides/uninstall-litmus.md:40]: Have added that at the bottom with the title `Remove the Litmus CRDs`

File: website/docs/user-guides/uninstall-litmus.md (hunk 2 above)
Comment (S-ayanide) [reply website/docs/user-guides/uninstall-litmus.md:48]: This has it own seperate section

File: website/docs/user-guides/uninstall-litmus.md (hunk 3 above)
Comment (S-ayanide) [reply website/docs/user-guides/uninstall-litmus.md:57]: Have added the subscriber

**Review by imrajdas — APPROVED** [review]

**Pull request merged**